harvard-university-of-worcester.csl

```diff
@@ -25,7 +25,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
+      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="20" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
```

Comment:
and here please: They don't do anything and setting them in bibliography/citation respectively will make it much easier for others to work with this style